sdk/src/androidTest/java/com/mapbox/maps/TestUtils.kt

```diff
@@ -0,0 +1,31 @@
+package com.mapbox.maps
+
+import android.view.View
+import androidx.core.view.children
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.TimeoutException
+
+internal fun MapView.hasChildView(view: View): Boolean {
+  children.forEach { v ->
+    if (v == view) {
+      return true
+    }
+  }
+  return false
+}
+
+internal fun MapView.getChildViewIndex(view: View): Int {
+  children.forEachIndexed { i, v ->
+    if (v == view) {
+      return i
+    }
+  }
+  return -1
+}
+
+internal fun CountDownLatch.throwExceptionOnTimeoutMs(timeoutMs: Long) {
```

Comment:
```suggestion
internal fun CountDownLatch.throwExceptionOnTimeoutMs(timeoutMs: Long = SOME_DEFAUL_VALUE) {
```
5_000 ms maybe

Reply:
don't want to expose this constant from view annotation file, let's leave as is. other tests most likely may have other values.

Reply:
this extension can be helpful for all maps tests.
and if other tests want to define their own value, they can easily pass it